dev-libs/benchmark/benchmark-1.3.0.recipe

```diff
@@ -1,26 +1,28 @@
 SUMMARY="A microbenchmark support library"
 DESCRIPTION="A library to support the benchmarking of functions, similar to unit-tests."
-HOMEPAGE="https://github.com/google/benchmark/"
+HOMEPAGE="https://github.com/google/benchmark"
 COPYRIGHT="2015-2017 Google Benchmark developers"
 LICENSE="Apache v2"
 REVISION="1"
 SOURCE_URI="https://github.com/google/benchmark/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="3dcc90c158838e2ac4a7ad06af9e28eb5877cf28252a81e55eb3c836757d3070"
+CHECKSUM_SHA256="f19559475a592cbd5ac48b61f6b9cedf87f0b6775d1443de54cfe8f53940b28d"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
```

Comment:
We like to have an empty line before `ARCHITECTURES`. Thanks :)

Source: [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)

Reply:
So much to keep track of!  Thanks for your patience!

Reply:
No problem :)
Thank **_you_** for contributing!